### ISSUE TITLE
Can not catch despite Pokemon bag is not full

### ISSUE THREAD

**Issue by hanseartic** [OpenPoGo/OpenPoGoBot#245]
### Expected Behavior

Bot catches Pokémon until bag is full
### Actual Behavior

Bot does not catch despite some free slots in Pokémon bag
### Steps to Reproduce

Start up bot with a almost filled Pokémon bag and restart after it tells you that the bag is full:

```
[2016-08-09 15:30:54] Web client connected
[2016-08-09 15:30:55] [+] Login to Pokemon Go successful.
[2016-08-09 15:30:55] [#]
[2016-08-09 15:30:55] [#] Username: xxx
[2016-08-09 15:30:55] [#] Acccount creation: 2016-07-27 09:43:48.785000
[2016-08-09 15:30:55] [#] Bag storage: 352/350
[2016-08-09 15:30:55] [#] Pokemon storage: 241/250
[2016-08-09 15:30:55] [#] Stardust: 105458
[2016-08-09 15:30:55] [#] Pokecoins: 0
[2016-08-09 15:30:55] [#] Poke Balls: 49
[2016-08-09 15:30:55] [#] Great Balls: 83
[2016-08-09 15:30:55] [#] Ultra Balls: 16
[2016-08-09 15:30:55] [#] -- Level: 21
[2016-08-09 15:30:55] [#] -- Experience: 285500
[2016-08-09 15:30:55] [#] -- Experience until next level: 49500
[2016-08-09 15:30:55] [#] -- Pokemon captured: 894
[2016-08-09 15:30:55] [#] -- Pokestops visited: 982
[2016-08-09 15:30:55] [#]
[2016-08-09 15:30:55] [x] Starting PokemonGo Bot....
[2016-08-09 15:30:56] [Catch] Pokemon bag is full; cannot catch.
```

Probably the catch plugin is checking the wrong storage for the item-count (note, that _Bag storage_ is overfilled)
### Other Information

OS: OSX Yosemite
Git Commit: ae031f1

**Comment by ajurna** [comment]
eggs count as pokemon. that is where the discrepancy is. this should be closed.

**Comment by niicojs** [comment]
we should update pokemon count to include eggs

**Comment by ajurna** [comment]
here is a pull request for that.
https://github.com/OpenPoGo/OpenPoGoBot/pull/249

**Comment by wchill** [comment]
Closed by #253
